scipy/linalg/tests/test_lapack.py

```diff
@@ -2923,3 +2924,50 @@ def test_ptsvx_NAG(d, e, b, x):
     df, ef, x_ptsvx, rcond, ferr, berr, info = ptsvx(d, e, b)
     # determine ptsvx's solution and x are the same.
     assert_array_almost_equal(x, x_ptsvx)
+
+
+@pytest.mark.parametrize('lower', [False, True])
+@pytest.mark.parametrize('dtype', DTYPES)
+def test_pptrs_pptri_pptrf_ppsv_ppcon(dtype, lower):
+    seed(1234)
+    atol = np.finfo(dtype).eps*100
+    # Manual conversion to/from packed format is feasible here.
+    n, nrhs = 10, 4
+    a = generate_random_dtype_array([n, n], dtype=dtype)
+    b = generate_random_dtype_array([n, nrhs], dtype=dtype)
+
+    a = a.conj().T + a + np.eye(n, dtype=dtype) * dtype(5.)
+    if lower:
+        inds = ([x for y in range(n) for x in range(y, n)],
+                [y for y in range(n) for x in range(y, n)])
+    else:
+        inds = ([x for y in range(1, n+1) for x in range(y)],
+                [y-1 for y in range(1, n+1) for x in range(y)])
+    ap = a[inds]
+    ppsv, pptrf, pptrs, pptri, ppcon = get_lapack_funcs(('ppsv', 'pptrf',
+                                                         'pptrs', 'pptri',
+                                                         'ppcon'), dtype=dtype)
```

Comment:
If you add `ilp64="preferred"` here, also that path gets tested on the CI. (Or, parametrize the test; `scipy.linalg.blas.HAS_ILP64` is the relevant enable flag.)

Reply:
Ah sure. This will take some time for me to get used to this.